### PR TITLE
NEWS: 1.13.1 update

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
update NEW for UCX v1.13.1 release

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -11,6 +11,16 @@
 ### Features:
 ### Bugfixes:
 
+## 1.13.1 (August 22, 2022)
+#### Bugfixes
+* Fixed Flow Control protocol in DC transport
+* Fixed reordering of pending operations in DC transport
+* Improved relaxed order recognition in IB transports
```
Comment (yosefe) [inline NEWS:18]: recognition -> detection

File: NEWS [hunk 2]
```diff
@@ -11,6 +11,16 @@
 ### Features:
 ### Bugfixes:
 
+## 1.13.1 (August 22, 2022)
+#### Bugfixes
+* Fixed Flow Control protocol in DC transport
```
Comment (yosefe) [inline NEWS:16]: flow control smallcase

File: NEWS [hunk 3]
```diff
@@ -11,6 +11,16 @@
...
+* Fixed reordering of pending operations in DC transport
+* Improved relaxed order recognition in IB transports
+* Synchronized build configuration and IB ops references
```
Comment (yosefe) [inline NEWS:19]: Should be phrased as "Fixed ... "

File: NEWS [hunk 4]
```diff
@@ -11,6 +11,16 @@
...
+* Updated wireup lane bandwidth calculation
```
Comment (yosefe) [inline NEWS:20]: Fixed bandwidth calculation during wireup phase

File: NEWS [hunk 5]
```diff
@@ -11,6 +11,16 @@
...
+* Do not check IP local port range for server port in TCP transport
```
Comment (yosefe) [inline NEWS:21]: Fixed TCP transport server port selection

**Comment by shamisp** [comment]
should be RC1

File: NEWS [hunk 6]
```diff
@@ -11,6 +11,16 @@
 ### Features:
 ### Bugfixes:
 
+## 1.13.1 (August 22, 2022)
+#### Bugfixes
+* Fixed flow control protocol in DC transport
+* Fixed reordering of pending operations in DC transport
+* Improved relaxed order detection in IB transports
```
Comment (shamisp) [inline NEWS:18]: Improved -> Fixed

**Review by yosefe — APPROVED** [review]
LGTM except what Pasha commented

**Comment by yosefe** [comment]
LGTM except the build issue

**Review by yosefe — APPROVED** [review]

**Review by shamisp — APPROVED** [review]

**Comment by evgeny-leksikov** [comment]
@tonycurtis could you please review?

**Review by tonycurtis — APPROVED** [review]

**Pull request merged**